src/Entity/ElectionRound.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+namespace AppBundle\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Bridge\Doctrine\Validator\Constraints\UniqueEntity;
+use Symfony\Component\Validator\Constraints as Assert;
+
+/**
+ * @ORM\Entity
+ * @ORM\Table(name="election_rounds")
+ *
+ * @UniqueEntity({"election", "label"})
+ */
+class ElectionRound
+{
+    /**
+     * @var int|null
+     *
+     * @ORM\Id
+     * @ORM\Column(type="integer")
+     * @ORM\GeneratedValue(strategy="IDENTITY")
```

Comment:
Don't we prefer a UUID instead ?

Reply:
Not for now. This is a first draft, many PR will compose this feature and the model may change during the process.

Reply:
I am not convinced and don't really understand why it's not done now but ok :D

Reply:
UUID has a performance cost and is an "extra" id because we always keep an internal primary ke. We should use it only when exposing resources via url to avoid leaking the "real" id.

I'm not convinced for now that we should add it on these entities.

Reply:
Unrelated to this PR but what about UUID only instead of having both ? It looks like the guys from Doctrine does not like auto generated ids :D

Reply:
I guess because performances with UUID only are worse. The choice was originally made by @tgalopin and @hhamon, maybe they can enlighten us more?

Reply:
@B-Galati you should never rely on UUID for foreign keys, they are using much more memory in indexes. On the other hand, you should also never rely on internal integers for public pages because the can be counted so they enable easy scraping. To get the best from both worlds, you can have both an ID and a UUId: the UUID for WHERE queries and the ID for joins.

Reply:
Note that in this specific PR, I don't see any usage of a WHERE query for elections, so adding an UUID does not seem necessary, I agree with @HeahDude

Reply:
Thanks @tgalopin ;-)